src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/config/JurisdictionToUserMapping.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.reform.bulkscan.endtoendtests.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import uk.gov.hmcts.reform.bulkscan.endtoendtests.ccd.Credential;
+
+import java.util.AbstractMap;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import static java.util.Map.Entry;
+import static java.util.stream.Collectors.toMap;
+
+@ConfigurationProperties(prefix = "idam")
+public class JurisdictionToUserMapping {
```

Comment:
how many users to we need?

Reply:
only bulkscan. this can be done simpler, I agree